firefox_puppeteer/api/l10n.py

```diff
@@ -40,8 +40,8 @@ def get_entity(self, dtd_urls, entity_id):
 
         with self.marionette.using_context('chrome'):
             value = self.marionette.execute_script("""
-                var parser = Cc["@mozilla.org/xmlextras/domparser;1"]
-                              .createInstance(Ci.nsIDOMParser);
+                var parser = Components.classes["@mozilla.org/xmlextras/domparser;1"]
+                              .createInstance(Components.interfaces.nsIDOMParser);
```

Comment:
This has to be changed because `Cc` and `Ci` is not available for window types other then browser.